examples/testprograms/http/autokitteh.yaml

```diff
@@ -15,6 +15,8 @@ project:
           value: "moo"
         - name: PIG
           value: "oink"
+    - name: gemini2
```

Comment:
just to ensure that this addition is related to this PR and should be in examples?

Reply:
The name "examples" is bad IMO, this code is mostly for tests.

Reply:
yeah, i will drop this change. it's more for manual testing. maybe it shouldn't belong in the repo at all.

Reply:
done